react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+const colorChange = {
+  3: 'purple',
+  6: 'blue',
+  9: 'red',
+  12: 'orange',
+  15: 'yellow',
+  18: 'white'
+};
+
+let thisManyClicks = 0;
+
```

Comment:
We don't want to use a global variable because that would cause conflicts between multiple instances of the same component.